openscm/core.py

```diff
@@ -256,20 +275,28 @@ def get_writable_timeseries_view(
         parameter = self._get_or_create_parameter(
             name, self._get_or_create_region(region)
         )
-        timeseries = Timeseries(start_time, period_length)
-        parameter.attempt_write(unit, ParameterType.TIMESERIES, timeseries)
-        return WritableTimeseriesView(parameter, unit, timeseries)
-
-    def get_parameter_info(self, name: Tuple[str], region: Tuple[str]) -> ParameterInfo:
+        parameter.attempt_write(unit, timeseries_type, time_points)
+        return WritableTimeseriesView(
+            parameter,
+            unit,
+            time_points,
+            timeseries_type,
+            interpolation_type,
+            extrapolation_type,
+        )  # WritableTimeseriesView
```

Comment:
```suggestion
        )
```

Reply:
Unfortunately, pylint only checks for duplicate lines, not statements. These comments are needed to avoid false positives of duplicate code (see last commit).